### PR TITLE
Fix stories and add incumbency

### PR DESCRIPTION
**Pull request by kangarang**
- fix and clean up several stories where context is necessary
- add component to view incumbency status of slate submitters

### CODE REVIEW

**Comment by kangarang** [comment]
do we want the organization to be displayed or the `firstName + lastName` (`owner`)?

**Comment by kangarang** [comment]
needs 1 more commit to handle titles for proposal cards

File: client/components/Card.tsx [hunk 1]
```diff
@@ -146,13 +156,32 @@ const Card: React.FunctionComponent<ICardProps> = props => {
...
     >
-      <div className="flex">
+      <Flex>
         {/* GRANT | PENDING TOKENS */}
         <Tag status={''}>{props.category.toUpperCase()}</Tag>
         {props.status && <Tag status={props.status}>{props.status}</Tag>}
-      </div>
+      </Flex>
+
+      {props.type === GRANT_SLATE && props.incumbent ? (
```
Comment (akuanti) [inline client/components/Card.tsx]: Is there a reason this is only for grant slates?

Comment (kangarang) [reply client/components/Card.tsx]: thanks. updated to include all types of 'SLATE's

**Pull request merged**